src/containers/AuthenticationPage/AuthenticationPage.js

```diff
@@ -489,6 +547,7 @@ export const AuthenticationPageComponent = props => {
               idpAuthError={authError}
               signupError={signupError}
               confirmError={confirmError}
+              userFields={userFields}
```

Comment:
This is passed quite deeply here. Perhaps it would make more sense to get the config directly where it's needed:
`const config = useConfiguration();`

Reply:
Roger, set useConfiguration() to the parent components of each form!